tools/common_lib/src/dnnl_utils.h

```diff
@@ -17,6 +17,7 @@ struct binding_t
     DataType dt = DataType::eCount;
     DataLayout layout = DataLayout::eCount;
     TensorShape shape;
+    bool transposed = false;
```

Comment:
"transposed" is not a valid flag for binding object.
Binding is a generic thing, transpose is specific to GEMM, so such concepts shouldn't be mix and matched in single structure.
You just should pass layout (line 18) which is transposed when the bindings are created.